Sources/HummingbirdAuth/Sessions/Request+session.swift

```diff
@@ -12,13 +12,11 @@
 //
 //===----------------------------------------------------------------------===//
 
-import ExtrasBase64
-import Foundation
 import Hummingbird
 import HummingbirdFoundation
 
 /// Manage session ids and associated data
-public struct SessionManager: Sendable {
```

Comment:
This should never have been set to conform to `Sendable`. Given it was only ever accessed via `HBRequest.session` I think it is safe to remove this conformance.